iot-hub/Quickstarts/read-d2c-messages/package.json

```diff
@@ -17,6 +17,7 @@
   },
   "homepage": "https://github.com/Azure-Samples/azure-iot-samples-node#readme",
   "dependencies": {
-    "@azure/event-hubs": "^1.0.3"
+    "@azure/event-hubs": "^1.0.3",
+    "request": "^2.88.0"
```

Comment:
Please provide
 -  the version of node.js on the system (node -v)
- the version of npm on the system (npm -v)
- the error that you are seeing while installing the sample.

Reply:
```node``` -> v8.10.0

```npm``` -> 6.4.1

Stack trace ->
```javascript
module.js:549
    throw err;
    ^

Error: Cannot find module 'request'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/angelonakos/Documents/GitHub/azure-iot-samples-node-master/iot-hub/Quickstarts/read-d2c-messages/node_modules/@azure/event-hubs/node_modules/ms-rest-azure/lib/credentials/msiVmTokenCredentials.js:12:17)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
```

Reply:
If I run ```npm install requests --save`` the ```requests``` module is added as a dependency inside the ```package.json```.

Running ```node ReadDeviceToCloudMessages.js``` will now NOT resolve into the aforementioned error.